src/esp/metadata/managers/LightLayoutAttributesManager.cpp

```diff
@@ -98,23 +98,27 @@ void LightLayoutAttributesManager::setLightInstanceValsFromJSONDoc(
   // jsonConfig here holds the JSON description for a single light attributes.
   // set position
   bool posIsSet = io::jsonIntoConstSetter<Magnum::Vector3>(
-      jsonConfig, "position",
-      std::bind(&LightInstanceAttributes::setPosition, lightAttribs, _1));
+      jsonConfig, "position", [lightAttribs](auto&& PH1) {
+        lightAttribs->setPosition(std::forward<decltype(PH1)>(PH1));
+      });
```

Comment:
This is great! Thanks Aaron. I'm going through these and will rewrite the lamdas to be a bit more semantically meaningful.